From 1fadac2354f700b293bccade044dc74be2ccf13a Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Fri, 11 Nov 2016 15:48:41 -0800
Subject: [PATCH] MANTA-2953 minfo not present in marlin jobs Reviewed by:
 Brian Bennett <brian.bennett@joyent.com> Reviewed by: Robert Mustacchi
 <rm@joyent.com> Approved by: Robert Mustacchi <rm@joyent.com>

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index dce7b7f..b7e08e7 100644
--- a/Makefile
+++ b/Makefile
@@ -296,8 +296,8 @@ $(PROTO_BOOT_ROOT)/setup.sh:
 # define a target that re-invokes "make" again after node_modules has been
 # installed.
 #
-USER_TOOLS_MANTA = mfind mget mjob mln mlogin mls mmd5 mmkdir mput mrm mrmdir \
-    msign muntar
+USER_TOOLS_MANTA = mchattr mchmod mfind mget minfo mjob mln mlogin mls mmd5 \
+    mmkdir mput mrm mrmdir msign muntar
 USER_TOOLS_MCB = maggr mcat mpipe msplit mtee
 PROTO_USER_TOOLS_ROOT = $(PROTO_MARLIN_ROOT)/ubin
 
-- 
2.21.0

